src/jaxsim/terrain/terrain.py

```diff
@@ -145,7 +147,7 @@ def height(self, x: jtp.FloatLike, y: jtp.FloatLike) -> jtp.Float:
         D = -C * self._height
 
         # Invert the plane equation to get the height at the given (x, y) coordinates.
-        return jnp.array(-(A * x + B * y + D) / C).astype(float)
+        return jnp.array(-(A * x + B * y + D) / (C + 1e-9)).astype(float)
```

Comment:
Mmh $C=0$ is a vertical plane, right? Is this something that happened?

Reply:
Yes, correct! A possible alternative is to use `jaxsim.exception` to raise a ValueError in case `C = 0`. If you prefer this alternative I can amend the commit

Reply:
The small epsilon won't affect that much but if we can avoid it, it would be preferable. Raising an exception is a good option for now.

Reply:
Fixed with rebase in https://github.com/ami-iit/jaxsim/pull/272/commits/99e6ecb3736fd03d6bb772fc3b25d0dfe2c3e979